Jenkinsfile-Release

```diff
@@ -88,9 +121,29 @@ String verifyIndexYamlAndTarFile(String chart) {
   String repositoryVersion = repositoryIndexFile.entries[chart][0].version
 
   if (!version.equals(repositoryVersion)) {
-    error "Released version: ${version} is different " + 
+    error "Released version: ${version} is different " +
           "from helm repository version: ${repositoryVersion}"
   }
-  
+
   return repositoryVersion
 }
+
+String calculateChartVersion(final String chartVersion, final String appVersion) {
+  if (chartVersion) {
+    return chartVersion
+  }
+
+  if (! appVersion) {
+    error 'Failed to calculate chartVersion with no appVersion.'
+  }
+
+  final versionParts = appVersion.split(/\./)
```

Comment:
should be able to utilize https://docs.sonatype.com/display/CDI/parseVersionString